src/Juvix/Backends/Michelson/Parameterisation.hs

```diff
@@ -23,7 +23,7 @@ import Prelude (String)
 -- TODO: Add rest of primitive values.
 -- TODO: Add dependent functions for pair, fst, snd, etc.
 typeOf ∷ PrimVal → NonEmpty PrimTy
-typeOf (PrimConst v) = PrimTy (M.Type (constType v) "") :| []
+typeOf (Constant v) = PrimTy (M.Type (constType v) "") :| []
```

Comment:
Given the `V.Value` `Int` / `Nat` / `Mutez` behaviour above; we may need to refactor this in a follow-up - https://github.com/cryptiumlabs/juvix/issues/326.